cg/meta/workflow/nf_analysis.py

```diff
@@ -278,5 +280,8 @@ def get_deliverables_for_case(self, case_id: str) -> PipelineDeliverables:
                 file[deliverable_field] = file[deliverable_field].replace(
                     "PATHTOCASE", str(self.get_case_path(case_id=case_id))
                 )
+            if file["tag"] in optional_file_tags and not Path(file["path"]).exists():
```

Comment:
Does the set of tags need to be included as a parameter of the function? If not, you could import the set with `from cg.constants.rnafusion import RNAFUSION_OPTIONAL_FILE_TAGS` and do
```suggestion
            if file["tag"] in RNAFUSION_OPTIONAL_FILE_TAGS and not Path(file["path"]).exists():
```
and you don't need to modify the signature of the function. Otherwise, you will have to import `RNAFUSION_OPTIONAL_FILE_TAGS` every time you use the function, while with this change you only need to import it once. Unless the tags will be different every time you call the function

Reply:
I will leave as it is